page_test.go

```diff
@@ -892,6 +894,51 @@ func TestPageElementFromObjectErr(t *testing.T) {
 	g.Err(p.ElementFromObject(obj.Object))
 }
 
+func TestPageTriggerFavicon(t *testing.T) {
+	g := setup(t)
+
+	// test browser in no-headless mode with an error
+	{
+		var l *launcher.Launcher
+		if runtime.GOOS == "darwin" {
```

Comment:
It will make testing on macOS flash a browser. Please use `g.mc.stub` here to mock the return value of `Browser.isHeadless`.